mappings/net/minecraft/world/ScheduledTick.mapping

```diff
@@ -13,6 +13,7 @@ CLASS bib net/minecraft/world/ScheduledTick
 		ARG 2 t
 		ARG 3 time
 		ARG 5 priority
+	METHOD a getComparator ()Ljava/util/Comparator;
```

Comment:
`comparing` (called as `ScheduledTick.comparing()`). Used in a similar fashion to utilities such as `Comparator.comparing(keyExtractor, keyComparator)`.